### PR TITLE
GitHub Action: support -a and -c

### PR DESCRIPTION
**Pull request by filip-zyzniewski**
Missed in
f8458c502b04014ba275433452732fb64de128f6
and
6e97682eb0ff658231fed2369bebceacbc91be44
.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 inputs:
+  ansi:
+    description: whether to keep ANSI escape sequences in the commit message
+    default: 'false'
   branch:
     description: which branch to pull from and push to
   command:
     description: shell command to be run by gitolize
     required: true
+  commit-sha:
```
Comment (pbzdyl) [inline action.yml]: What about a bit longer input name that might be more readable in places this action is used and the its description is not available?
```suggestion
  print-commit-sha:
```

File: action.yml [hunk 2]
```diff
@@ -1,11 +1,17 @@
 name: Gitolize
 description: Wrap a shell command with gitolize
 inputs:
+  ansi:
```
Comment (pbzdyl) [inline action.yml]: What about a bit longer input name that might be more readable in places this action is used and the its description is not available? (or something shorter than proposed but still hinting what the context is)
```suggestion
  keep-ansi-in-commit-message:
```

**Review by pbzdyl — DISMISSED** [review]

File: action.yml (hunk 1 above)
Comment (filip-zyzniewski) [reply action.yml]: Done in efb7c81e080558b9859da783bf75d4284667893d.

File: action.yml (hunk 2 above)
Comment (filip-zyzniewski) [reply action.yml]: Done in 583316659c366007d104ef7abf21adb1ead1ef71.

**Review by pbzdyl — APPROVED** [review]

**Pull request merged**